MapboxAndroidDemo/src/main/res/values/strings.xml

```diff
@@ -95,7 +96,7 @@
     <string name="activity_plugins_building_plugin_title">Display buildings in 3D</string>
     <string name="activity_plugins_location_plugin_title">Show a user\'s location</string>
     <string name="activity_mas_simplify_polyline_title">Simplify a polyline</string>
-    <string name="activity_mas_map_matching_title">Map Matching</string>
+    <string name="activity_mas_map_matching_title">Map matching</string>
```

Comment:
No need to change this, unrelated and `M` is fine by looking at https://www.mapbox.com/api-documentation/.